css/selectors/placeholder.json

```diff
@@ -26,11 +26,13 @@
             ],
             "edge": {
               "prefix": "-webkit-input-",
-              "version_added": true
+              "version_added": true,
+              "notes": "Also supports the <code>-ms-input-</code> prefix."
```

Comment:
I don't think we need to hide this data in a note. How about an array like this?

```
[
  {
    "prefix": "-ms-input-",
    "version_added": true
  },
  {
    "prefix": "-webkit-input-",
    "version_added": true
  }
]
```

The same would work for `edge_mobile`

Reply:
Sure thing!  Changes coming right up.  :wink: